### PR TITLE
Updated regtest setup guide

### LINKED ISSUE #338: Update the regtest set-up guide
Steps:
1) Update the steps(2 points)
2) Try on a fresh OS install(2 points)

### PR DESCRIPTION
**Pull request by hackrush01**
Closes #338

### CODE REVIEW

File: content/faq/regtest-setup-guide.md [hunk 1]
```diff
@@ -3,156 +3,188 @@ title: How do I setup an isolated regtest network to test LBRY?
...
 ```bash
-cd lbry
-pip2 install -r requirements.txt
-pip2 install .
+cd lbryum-server
+sudo pip2 install -r requirements.txt
+```
+
+If you're not running debian/*buntu, or a derivative of those, you need to edit the `configure` file a bit. On line 11, remove the `apt-get` line and manually install the required packages. On line 51, change `adduser` to `useradd` and on the same line, change `--disabled-password` to `-p !`.
+```bash
```
Comment (kaykurokawa) [inline content/faq/regtest-setup-guide.md]: I assume this is instructions for mac OS? Maybe we can just add this changed file as a new file in the lbryum-server repo?

File: content/faq/regtest-setup-guide.md [hunk 2]
```diff
@@ -3,156 +3,188 @@ title: How do I setup an isolated regtest network to test LBRY?
...
+## Why use a regtest server
+A regtest server provides for a way to instantly generate blocks so that transactions can be instantaneous, so ultimately no waiting for confirmations from the blockchain. Also, no problem if you accidently corrupt your wallet, since no real funds are lost! Just delete the files and setup a new one.
+
+## Setup
 
 To begin setting up the network, there are a few things you need.
 
-Note that you need a Linux distribution to run all this. A virtual machine is
-fine.
+Note that you need a Linux distribution to run all this. A virtual machine is fine.
```
Comment (kaykurokawa) [inline content/faq/regtest-setup-guide.md]: should also work on Mac , 
Can you also state here what distribution you used to test this instruction. I.e, 
"These instructions specifically were tested on Ubuntu version XX.XX "

File: content/faq/regtest-setup-guide.md (hunk 1 above)
Comment (hackrush01) [reply content/faq/regtest-setup-guide.md]: I don't have it available, it was already in the previous version of the guide so I let it remain there. But it seems these are the instructions for running it on ArchLinux and similar OSes.

**Comment by kaykurokawa** [comment]
lgtm

**Pull request merged**